sections/futures/Trade/DepositMarginModal.tsx

```diff
@@ -57,46 +51,36 @@ const DepositMarginModal: React.FC<DepositMarginModalProps> = ({
 	);
 
 	const ethPriceRate = React.useMemo(
-		() => getExchangeRatesForCurrencies(exchangeRates, Synths.sETH, selectedPriceCurrency.name),
+		() => newGetExchangeRatesForCurrencies(exchangeRates, Synths.sETH, selectedPriceCurrency.name),
 		[exchangeRates, selectedPriceCurrency.name]
 	);
 
-	const gasPrice = ethGasPriceQuery?.data?.[gasSpeed]
-		? parseGasPriceObject(ethGasPriceQuery?.data?.[gasSpeed])
-		: null;
+	const gasPrice = ethGasPriceQuery.data != null ? ethGasPriceQuery.data[gasSpeed] : null;
+
+	const depositTxn = useSynthetixTxn(
+		`FuturesMarket${market?.substring(1)}`,
```

Comment:
Is this stripping the `s` from `sBTC`? If so, we should be careful because future markets will not have the `s`.